src/Search.php

```diff
@@ -769,7 +769,8 @@ public static function getOptionNumber($itemtype, $field)
      **/
     public static function &getOptions($itemtype, $withplugins = true)
```

Comment:
```suggestion
    public static function getOptions($itemtype, $withplugins = true)
```

Reply:
Shouldn't the method signature remain the same since it is used by plugins and would be deprecated anyways?

Reply:
We should not try to preserve BC here. We should just change method signature and indicates it in the changelog.